src/table/misc_settings.ini

```diff
@@ -308,5 +308,14 @@ min      = ZOOM_LVL_MIN
 max      = ZOOM_LVL_OUT_4X
 cat      = SC_BASIC
 
+[SDTG_VAR]
+name     = ""font_zoom""
+type     = SLE_UINT8
+var      = _font_zoom
+def      = ZOOM_LVL_OUT_4X
```

Comment:
The default value should be ZOOM_LVL_MIN, it was surprising to see everything so big on starting the game.

Reply:
Good catch, thanks! Just updated.